code/modules/surgery/transorbital_lobotomy.dm

```diff
@@ -0,0 +1,63 @@
+/datum/surgery/transorbital_lobotomy
+	name = "Transorbital Lobotomy"
+	steps = list(/datum/surgery_step/pick, /datum/surgery_step/drive, /datum/surgery_step/tlobotomize, /datum/surgery_step/close)
+	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_PRECISE_EYES)
+
+/datum/surgery/transorbital_lobotomy/can_start(mob/user, mob/living/carbon/target)
+	. = ..()
+	if(!.)
+		return
+	return target.getorganslot(ORGAN_SLOT_BRAIN) ? TRUE : FALSE
+
+
+/datum/surgery_step/tlobotomize
```

Comment:
why'd you name this path "tlobotomy" if there's no "lobotomy" path to compete with anymore?